pr_check.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# --------------------------------------------
+# Options that must be configured by app owner
+# --------------------------------------------
+APP_NAME="ingress"  # name of app-sre "application" folder this component lives in
+COMPONENT_NAME="insights-storage-broker"  # name of app-sre "resourceTemplate" in deploy.yaml for this component
+IMAGE="quay.io/cloudservices/insights-storage-broker"  
+
+IQE_PLUGINS="ingress"
+IQE_MARKER_EXPRESSION="smoke"
+IQE_FILTER_EXPRESSION=""
+
+
+# Install bonfire repo/initialize
+CICD_URL=https://raw.githubusercontent.com/RedHatInsights/bonfire/master/cicd
+curl -s $CICD_URL/bootstrap.sh -o bootstrap.sh
+source bootstrap.sh  # checks out bonfire and changes to "cicd" dir...
+
+source build.sh
+source deploy_ephemeral_env.sh
+source smoke_test.sh
```

Comment:
I suggest we comment out smoke_test.sh and replace it with something similar to what I did for host inventory:

https://github.com/redhatinsights/insights-host-inventory/blob/fd2354eed2aa811bcfb012679e9d0456c8d5ea6b/pr_check.sh#L41-L48